ember/app/ui/components/identity-members/component.js

```diff
@@ -21,63 +20,32 @@ export default class IdentityMembersComponent extends Component {
     return this.pageNumber < this.totalPages;
   }
 
-  @restartableTask
-  *fetchMembers() {
-    const membershipResponse = yield this.store.query(
-      "membership",
+  get locale() {
+    return this.intl.locale[0];
+  }
+
+  _members = trackedFunction(this, async () => {
+    const membersResponse = await this.store.query(
+      "identity",
       {
         filter: { organisation: this.args.identity.id },
-        include: "role,identity",
         page: {
           number: this.pageNumber,
           size: this.pageSize,
         },
       },
       { adapterOptions: { customEndpoint: "org-memberships" } }
     );
+    this.totalPages = membersResponse.meta.pagination?.pages;
 
-    this.totalPages = membershipResponse.meta.pagination?.pages;
-
-    membershipResponse.forEach((membership, i, a) => {
-      const identity = membership.identity;
-      const duplicateMembership = this.members.findBy(
-        "identity.id",
-        identity.get("id")
-      );
-
-      let title = membership.role?.get("title") ?? "";
-      if (title && a.filterBy("role.title").length !== i + 1) {
-        title += ",";
-      }
-
-      membership.roles = [
-        {
-          title,
-          inactive: membership.isInactive,
-        },
-      ];
-      if (duplicateMembership && membership.role.get("title")) {
-        duplicateMembership.roles = [
-          ...duplicateMembership.roles,
-          ...membership.roles,
-        ];
-      }
-
-      if (!duplicateMembership) {
-        this.members.push(membership);
-      }
-    });
-
-    this.members.forEach((member) => {
+    membersResponse.forEach((member) => {
       member.inactive = member.roles.every((role) => role.inactive);
+      this.members.push(member);
```

Comment:
Is this never reset if for instance your getting page 1 or something similar?

Reply:
You cannot reset to page 1 without reloading, as the UI only allow to load the next page